wp_api/src/api_client.rs

```diff
@@ -164,6 +146,18 @@ impl WpApiClient {
     }
 }
 
+// IMPORTANT: This type may be aggressively cloned, so all of its fields must be cheap to clone!
+#[derive(Clone, uniffi::Record)]
+pub struct WpApiClientDelegate {
+    pub auth_provider: Arc<WpAuthenticationProvider>,
+    pub request_executor: Arc<dyn RequestExecutor>,
+    pub middleware_pipeline: Arc<WpApiMiddlewarePipeline>,
+}
+
+pub trait IsRequestExecutor {
```

Comment:
Should this type be `IsWpApiClientDelegate`?

Reply:
I thought we might put some more request executor related methods here in the future which is why I picked this name, but I don't feel strongly about it 🤷‍♂️

Reply:
Addressed in 40ce5c7411da35a500dff4d7fec3178c4dfe0612.

Reply:
I missed one reference in my original commit which is addressed in 991a727d08d0ce97402a6656fc2573083aab211e.